### PR TITLE
Add support for Moxa EDS-G512E-8PoE

### PR DESCRIPTION
**Pull request by priiduonu**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/discovery/moxa-etherdevice.yaml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-mib: IF-MIB:MOXA-IKS6726A-MIB:MOXA-EDSG508E-MIB:MOXA-EDSP510A8POE-MIB
```
Comment (PipoCanaja) [inline includes/definitions/discovery/moxa-etherdevice.yaml]: You should keep IF-MIB here cause it is used for the description building : 
   'sensor_descr' => '{{ $ifDescr }} SFP module Transmit Power'
That explains why your SFP module transmit power lost its description.

Comment (PipoCanaja) [inline includes/definitions/discovery/moxa-etherdevice.yaml]: You should keep IF-MIB here cause it is used for the description building :
'sensor_descr' => '{{ $ifDescr }} SFP module Transmit Power'
That explains why your SFP module transmit power lost its de

Comment (priiduonu) [reply includes/definitions/discovery/moxa-etherdevice.yaml]: Will keep IF-MIB, thank You!

**Comment by PipoCanaja** [comment]
And following the add of the tests, you should check on Travis CI tests, they must succeed in order to merge the changes.

**Comment by PipoCanaja** [comment]
Thanx, It is much better, but it seems that your master is not up to date. Could you do: 
``` 
git checkout master
git pull origin master
git checkout newdevice/moxa-edsg512e8poe
git rebase master
git push origin newdevice/moxa-edsg512e8poe
```
That should ensure you are running your patch against an up to date code. 
PipoCanaja

**Pull request merged**